packages/itwin/breakdown-trees/package.json

```diff
@@ -0,0 +1,145 @@
+{
+  "name": "@itwin/breakdown-trees-react",
+  "version": "0.1.0",
+  "description": "Package for breakdown trees",
+  "keywords": [
+    "Bentley",
+    "iModel breakdown Tree",
+    "iModel.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iTwin/viewer-components-react/tree/master/packages/itwin/breakdown-trees"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/breakdown-trees-react.js",
+  "typings": "lib/breakdown-trees-react",
+  "scripts": {
+    "build": "npm run copy:assets && npm run -s build:cjs && npm run -s build:esm && tsc --sourceMap 1>&2",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs && npm run -s copy:cjs",
+    "build:esm": "tsc 1>&2 --outDir lib/esm --module ES2020 && npm run -s copy:esm",
+    "copy:cjs": "cpx \"./src/**/*.*css\" ./lib/cjs",
+    "copy:esm": "cpx \"./src/**/*.*css\" ./lib/esm",
+    "build-prod": "tsc 1>&2 && npm run copy:assets",
+    "rebuild": "npm run clean && npm run build",
+    "makegraph": "depcruise --exclude \"(node_modules|src/test)\" --output-type dot src > graphs/dependencygraph.svg",
+    "copy:assets": "cpx \"./src/assets/**/*\" ./lib/assets && cpx \"./src/**/*.scss\" ./lib && cpx \"./public/**/*\" ./lib/public",
```

Comment:
i think this line needs to get updated to call copy cjs & esm

Reply:
copy cjs and esm are called from the build script and removing that from copy:assets is causing test failure.So keeping it there.